test/unit/DCAFeeManager/dca-fee-manager.spec.ts

```diff
@@ -212,4 +287,13 @@ contract('DCAFeeManager', () => {
       });
     });
   }
+
+  function getPlatformBalance(address: string) {
+    return ethers.provider.getBalance(address);
+  }
+
+  async function setPlatformTokenBalance(recipient: { address: string }, amount: BigNumber) {
+    await ethers.provider.send('hardhat_setBalance', [recipient.address, ethers.utils.hexValue(amount)]);
+    return BigNumber.from(amount);
+  }
```

Comment:
To be completely honest `platformBalance` here is pretty confusing. Since that is what we call the fees on the `Hub`.

Reply:
You are right, it should be setProtocolTokenBalance. Platform balance is the fee on the hub. Protocol token is the chain's tokens

For v3 we should rethink names haha